contrib/eclair-cli_autocomplete.sh

```diff
@@ -0,0 +1,77 @@
+# eclair-cli_autocomplete.sh
+# 
+# To use this script, source it into your .bashrc
+# Use command `source /path/to/eclair-cli_autocomplete.sh`
+# This script provides command-line autocompletion for `eclair-cli.kexe`.
+# Once sourced in your shell (e.g., Bash), you can start typing a command
+# and use the <Tab> key to autocomplete or get suggestions for commands and options.
+# Tutorials followed: 
+# https://tldp.org/LDP/abs/html/
+# https://iridakos.com/programming/2018/03/01/bash-programmable-completion-tutorial
+# https://www.youtube.com/watch?v=emhouufDnB4
+
+#!/bin/bash
+
+_eclair_cli() {
+    local cur prev words cword
+    
+    # Initialize the completion environment. 
+    # `_init_completion` is a helper function provided by the Bash-completion package.
+    _init_completion || return
+
+    local commands="getinfo connect disconnect open rbfopen cpfpbumpfees close forceclose"
+    local common_opts="-p"
```

Comment:
Aren't you missing `--host`?

Reply:
Yes, I'll update it in the next commit.

Reply:
I don't see that commit?

Reply:
> I don't see that commit?

Sorry, it wasn't pushed. The commit is 97a8118511067ef105bc3c584ef15046f1e67b89.

Reply:
> I don't see that commit?

Here it is, 664828314e3ee0ba960711448e4a52acaaa45377. Sorry, it wasn't pushed earlier.